src/components/atoms/Checkbox/Checkbox.tsx

```diff
@@ -14,7 +14,7 @@ export interface CheckboxProps
   toggler?: boolean;
   containerClassName?: string;
   labelClassName?: string;
-  forwardRef?: (
+  forwardedRef?: (
```

Comment:
As called out in the previous review thread about this, the fix wasn't to rename this prop, the fix was to properly use the `forwardRef` function. Described in more detail in my previous comments.